script.js

```diff
@@ -20,7 +20,7 @@ function loadPage() {
   }
 
   function error(err) {
-    console.warn(`ERROR(${err.code}): ${err.message}`);
+    console.log(err);
```

Comment:
Would you like to change this back to "
    console.warn(`ERROR(${err.code}): ${err.message}`);"